wls-basisdaten-service/src/test/resources/http-client.env.json

```diff
@@ -1,8 +1,14 @@
 {
+  "docker": {
+    "token_type": "",
+    "auth_token": "",
+    "WLS_BASISDATEN_SERVICE_URL": "http://localhost:8205",
+    "SSO_URL": "http://kubernetes.docker.internal:8100"
+  },
   "nonDocker": {
     "token_type": "",
     "auth_token": "",
     "WLS_BASISDATEN_SERVICE_URL": "http://localhost:39151",
-    "SSO_URL": "http://kubernetes.docker.internal:8100"
+    "SSO_URL": "http://localhost:8100"
```

Comment:
Bitte reverten. So funktioniert der Service local nicht mehr mit Keycloak.

Der Token wird von `localhost` ausgestellt. Der Service hingegen checkt gegen `kubernetes.docker.internal`. Auch wenn es am Ende die gleichen Anwendungen sind wird es failen weil es "unterschiedliche" Hosts sind.

Reply:
Danke! erledigt in af2896a9fe2b8ca1f1b39e60a465180c18a1792d